docs/source/getting_started/index.md

```diff
@@ -134,7 +134,13 @@ client = (
 )  # or create_http_client() depending on the environment you picked
 
 # List available models
-models = client.models.list()
+
```

Comment:
I don't think this is needed either because the program is going to exit regardless and `print(e)` isn't even adding any better error handling?